src/main/java/io/appium/java_client/android/ListensToLogcatMessages.java

```diff
@@ -33,13 +33,13 @@
 public interface ListensToLogcatMessages extends ExecutesMethod {
     StringWebSocketClient getLogcatClient();
 
+
     /**
      * Start logcat messages broadcast via web socket.
-     * This method assumes that Appium server is running on localhost and
-     * is assigned to the default port (4723).
      */
     default void startLogcatBroadcast() {
-        startLogcatBroadcast("localhost", DEFAULT_APPIUM_PORT);
+        URL remoteAdress = ((RemoteWebDriver) this).getRemoteAddress();
```

Comment:
This might break the backward compatibility consider created another overriden method there, which allows to set the host and port in arguments

Reply:
I have trouble understanding what you mean. There is still the `default void startLogcatBroadcast(String host, int port)` if you want to set it manually.

Reply:
yes, I see now. Then such change would be a breaking one